frontend/src/metabase/core/components/Radio/Radio.tsx

```diff
@@ -55,6 +55,7 @@ export interface RadioProps<TValue extends Key, TOption = RadioOption<TValue>>
   showButtons?: boolean;
   onChange?: (value: TValue) => void;
   onOptionClick?: (value: TValue) => void;
+  wrap: boolean;
```

Comment:
Can we actually just pass the desired styles via emotion? See an example how [`DatasetFieldMetadataSidebar` styles `SelectButton's` nested component.](https://github.com/metabase/metabase/blob/91a87d814fa20125bacddf1d72ffd8e9dfe7a5f6/frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataSidebar.styled.jsx#L33). You'll need to assign these internal visual components to `Radio`, see [how `SelectButton` does that](https://github.com/metabase/metabase/blob/91a87d814fa20125bacddf1d72ffd8e9dfe7a5f6/frontend/src/metabase/core/components/SelectButton/SelectButton.tsx#L85)

Reply:
Makes sense to me, given the component API.